packages/api/tests/lib/rules/test-rulesHelpers.js

```diff
@@ -65,6 +80,21 @@ const getKinesisEventMappings = async () => {
   return await Promise.all(mappingPromises);
 };
 
+const deleteKinesisEventSourceMappings = async () => {
```

Comment:
So this is a refactor of the kinesis logic previously in `deleteOldEventSourceMappings` and moved here because it's only a test helper?

Reply:
Yes, that's correct.